kernel/src/fs/procfs/pid/stat.rs

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use core::fmt::Write;
+
+use crate::{
+    fs::{
+        procfs::template::{FileOps, ProcFileBuilder},
+        utils::Inode,
+    },
+    prelude::*,
+    Process,
+};
+/// Represents the inode at `/proc/[pid]/stat`.
+/// FIXME: Some fields are not implemented yet.
```

Comment:
Can we put a link here like you comment in front of this PR that can remind us which fields exactly we support or donot support and where should we go to check?

Reply:
Of course. I will add it in the latest version.